src/main/java/frc/team4909/robot/sensors/PhotoElectricSensors.java

```diff
@@ -0,0 +1,75 @@
+package frc.team4909.robot.sensors;
+import frc.team4909.robot.Robot;
+import edu.wpi.first.wpilibj.DigitalInput;
+public class PhotoElectricSensors{  //not sure if extends or implements
```

Comment:
`implements` means that there is some `Interface` that dictates what methods you must implement in your class, to conform to that `Interface`.  Below, you are calling `super` in the constructor, which indicates that there is, in fact, a superclass... which means that this class must `extend` some class (which becomes the superclass).